_pages/en/target-solution-delivery-model-strategy-areas-of-concerns.md

```diff
@@ -30,353 +32,293 @@ This document will be updated throughout the elaboration of the above strategy.
+The meritocracy approach recognizes the different levels of maturity between teams and, with this level of maturity, the level of trust the organization should allocate.
+We are currently looking at a meritocracy approach that favours and incentivize continuous learning for DevOps teams.
 
-#### 2.9.1. Summary <!-- omit in toc -->
+3. **Management of personnel**  
+Details on this piece has not yet been figured out and will need to be looked at closely.
+In the beginning, it is foreseen that we will have cross-functional teams (i.e., members from different functional specifications assembled as a single team), but the target state looks at embedded members with a shared skill set (skill set at the team level, not individually assigned to members).
+This target state is different than our current functional organization setting.
 
-The strategy looks at limiting the size of projects to Small and below.
-Is it expected that, should an initiative require multiple releases to production and passes the $2.5M threshold, to redo the Project Management Lifecycle for the subsequent releases?
+### 2.6. Risk to enterprise reference data model and standard compliance
 
-If so, this creates a risk of too much overhead for IT Teams to handle and may result in slowing delivery.
+#### 2.6.1. Summary <!-- omit in toc -->
 
-#### 2.9.2. Raised by stakeholder(s) <!-- omit in toc -->
+Moving towards a more decentralized and distributed means of decision-making presents a risk that incorrect decisions may occur, affecting data management across the enterprise.
+The risk involves duplication of data, incoherence to data consumers, and degradation of data quality.
+Over the years, IITB has spent efforts centralizing certain functions to achieve efficiencies and be able to do more with less.
 
-- BSIM
+In the case of the Data Analytics team, direct access to a database as opposed to an API (exposing data meant for consumption by a User Interface or another functional API) is necessary due to the broad nature of analytics and its need for vast amount of data.
+An API may be built, per product, for the sole purpose of accessing the database content but:
 
-#### 2.9.3. What is being done about this concern <!-- omit in toc -->
+1. It's unlikely a Product will have it built, as it will require more costs to do so
+2. Performance hits are expected
+3. The number of APIs to consume and reconstruct data will increase complexity of Data Analytics solutions
 
-TBD.
+#### 2.6.2. Raised by stakeholder(s) <!-- omit in toc -->
 
-Explore multi-gated (3 and 4) PMLC approach.
+- DAS
 
-### 2.10. Risk to capability-based planning ability
+#### 2.6.3. What is being done about this concern <!-- omit in toc -->
 
-#### 2.10.1. Summary <!-- omit in toc -->
+This concern is not addressed at the strategy level (guiding policy) but needs to be reminded during its implementation.
+The following two careful considerations are necessary:
```

Comment:
I changed to "two" as there were 2 listed below. Feel free to add another one if missing.